go.mod

```diff
@@ -1,51 +1,41 @@
 module github.com/status-im/status-console-client
 
+go 1.13
+
+replace github.com/ethereum/go-ethereum v1.9.5 => github.com/status-im/go-ethereum v1.9.5-status.4
+
+replace github.com/NaySoftware/go-fcm => github.com/status-im/go-fcm v1.0.0-status
+
+replace github.com/status-im/status-protocol-go v0.0.0 => github.com/status-im/status-protocol-go v0.2.3-0.20191011094108-77478f220d48
+
 require (
 	github.com/Microsoft/go-winio v0.4.12 // indirect
 	github.com/allegro/bigcache v1.2.1 // indirect
-	github.com/btcsuite/btcd v0.0.0-20190629003639-c26ffa870fd8 // indirect
 	github.com/cespare/cp v1.1.1 // indirect
 	github.com/dhui/dktest v0.3.1 // indirect
 	github.com/docker/distribution v2.7.1+incompatible // indirect
 	github.com/docker/go-units v0.4.0 // indirect
 	github.com/edsrzf/mmap-go v1.0.0 // indirect
-	github.com/elastic/gosigar v0.10.3 // indirect
-	github.com/ethereum/go-ethereum v1.8.27
+	github.com/ethereum/go-ethereum v1.9.5
 	github.com/fatih/color v1.7.0
 	github.com/fjl/memsize v0.0.0-20180929194037-2a09253e352a // indirect
-	github.com/golang-migrate/migrate/v4 v4.5.0 // indirect
-	github.com/golang/mock v1.3.1 // indirect
 	github.com/google/uuid v1.1.1
 	github.com/gorilla/mux v1.7.3 // indirect
-	github.com/influxdata/influxdb v1.7.7 // indirect
 	github.com/jroimartin/gocui v0.4.0
-	github.com/karalabe/hid v1.0.0 // indirect
-	github.com/lib/pq v1.1.1 // indirect
-	github.com/libp2p/go-eventbus v0.0.3 // indirect
-	github.com/libp2p/go-libp2p v0.2.0 // indirect
-	github.com/libp2p/go-libp2p-swarm v0.1.1 // indirect
-	github.com/libp2p/go-msgio v0.0.4 // indirect
 	github.com/mattn/go-colorable v0.1.2 // indirect
 	github.com/mattn/go-runewidth v0.0.4 // indirect
-	github.com/multiformats/go-multihash v0.0.6 // indirect
-	github.com/mutecomm/go-sqlcipher v0.0.0-20190227152316-55dbde17881f // indirect
 	github.com/nsf/termbox-go v0.0.0-20190624072549-eeb6cd0a1762 // indirect
 	github.com/opentracing/opentracing-go v1.1.0 // indirect
-	github.com/pborman/uuid v1.2.0 // indirect
 	github.com/peterbourgon/ff v1.2.0
 	github.com/pkg/errors v0.8.1
 	github.com/rjeczalik/notify v0.9.2 // indirect
 	github.com/sirupsen/logrus v1.4.2 // indirect
-	github.com/status-im/status-go v0.29.0-beta.3
+	github.com/status-im/status-go v0.34.0-beta.0.0.20191011121408-2fd615d0e317
 	github.com/status-im/status-nim v0.0.0-20190724023117-a5693e6e4820
-	github.com/status-im/status-protocol-go v0.0.0-20190624072549-57675d19152c73af930b1e8f93d298322cbfe95c
+	github.com/status-im/status-protocol-go v0.3.1-0.20191011120912-b115a9b0ae4f
```

Comment:
`v0.3.1` -- how come as the latest is https://github.com/status-im/status-protocol-go/releases/tag/v0.3.0.

Reply:
I just put the SHA1 there and ran `mod tidy`, not sure.